src/main/java/kmql/Engine.java

```diff
@@ -76,6 +77,29 @@ private void prepareRequiredTables(String sql) {
         }
     }
 
+    /**
+     * Show columns of the given table
+     * @param table the target table
+     * @return the list of column names
+     */
+    public List<String> showColumns(Table table) {
```

Comment:
1. I think this should be a function of `Database` class rather than `Engine` (`Database` = abstract representation of the "database", `Engine` = representation of interaction with user's inputs and outputs)
2. `listColumns()` or just `columns()` sounds much suitable for this as a java method I think (it doesn't "show", it returns the "list").